components/AutoCompleteInput.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react';
+import { StyleSheet, View, Text, TextInput} from 'react-native';
+import Location from '../classes/location';
+import Colors from '../constants/Colors';
+import indoorMap from '../constants/indoorTry';
+
+const styles = StyleSheet.create({
+container:{
+flex:1,
+
+},
+modalSugg: {
+  position:"absolute",
+  top:40,
+},
+
+myInput:{
+        width: 250,
+        height: 40,
+        borderColor: Colors.black,
+        backgroundColor: Colors.white,
+        borderWidth: 0.5,
+        padding: 10,
+
+  },
+  suggestions:{
+    zIndex:8,
+    width: 250,
+    height: 45,
+    borderColor: Colors.black,
+    backgroundColor: Colors.white,
+    borderWidth: 0.5,
+    padding: 5,
+}
+}
+);
+type autoStates={
+query:string;
+places:any;
+position:Location;
+indoor:any;
+indoorResults:any;
+lat:number;
+long:number;
+indoorX: number;
+indoorY:number;
+indoorID:string;
+type:string;
+};
+class Autocomplete extends Component<{myposition:Location,type:string},autoStates> {
```

Comment:
type for prop should be declared above (same as state)

Reply:
resolved